conf/tools/blacklisted

```diff
@@ -0,0 +1,9 @@
+#Blacklisted tools
+#Add the name of the tool as defined in its xml file to hide it in the "Tools" menu in the paparazi center.
+#Lines starting with # are comments and will be ignored.
+#
+#Examples: (Uncomment to hide them in the paparazzi center)
+#BluegigaUsbDongleScanner
+#BluegigaUsbDongle
+#IridiumDialer
+
```

Comment:
This file should not be under revision control, only `blacklisted_example`. Can you remove it, and even better add it to `.gitignore` (should have been done before).

Reply:
I would be very happy to add it to .gitignore, I commit this file way too often...